<ui version="4.0" >
 <class>KWGeneralFrameProperties</class>
 <widget class="QWidget" name="KWGeneralFrameProperties" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="textGroupBox" >
       <property name="title" >
        <string>If Text is Too Long for Frame</string>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="createNewPage" >
          <property name="text" >
           <string>Create a new page</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="resizeLastFrame" >
          <property name="text" >
           <string>Resize last frame</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="noExtraText" >
          <property name="text" >
           <string>Don't show the extra text</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="newPageGroupBox" >
       <property name="title" >
        <string>On New Page Creation</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="noFollowup" >
          <property name="text" >
           <string>Do not create a followup frame</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="reconnect" >
          <property name="text" >
           <string>Reconnect frame to current flow</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="placeCopy" >
          <property name="text" >
           <string>Place a copy of this frame</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="evenOdd" >
          <property name="text" >
           <string>Only on even/odd pages</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <widget class="QCheckBox" name="protectContent" >
     <property name="text" >
      <string>Protect content</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QCheckBox" name="isCopyOfPrevious" >
     <property name="text" >
      <string>Frame is a copy of the previous frame</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QCheckBox" name="allFrames" >
     <property name="text" >
      <string>Changes will be applied to all frames in frameset</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>411</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>noFollowup</sender>
   <signal>toggled(bool)</signal>
   <receiver>evenOdd</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>364</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel" >
     <x>347</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
